page-objects/pages/coursePages/CRSSCourseListPage.js

```diff
@@ -3,6 +3,9 @@
 const { CLIENT } = require("../../../shared-objects/servers")
 const eh = require("../../../runtime/helpers/elementHelpers");
 const wh = require("../../../runtime/helpers/waitHelpers");
+const courseData = require('../../../shared-objects/courseData');
+const firstLogin = require('../../../shared_steps/firstLogin.js');
+const elementHelpers = require('../../../runtime/helpers/elementHelpers.js');
 const { expect } = require("chai");
```

Comment:
expect doesn't need to be imported as globally available

Reply:
done